packages/slate-react/src/utils/find-range.js

```diff
@@ -51,21 +49,6 @@ function findRange(native, editor) {
   const focus = isCollapsed ? anchor : findPoint(focusNode, focusOffset, editor)
   if (!anchor || !focus) return null
 
```

Comment:
would it be prudent to leave in the IS_IE check?

Reply:
I tested the issue described in PR #1574 in IE11. Seems to work fine.

![ie11](https://user-images.githubusercontent.com/1549771/56356333-dd96f980-61d8-11e9-8b37-d53538a7b69b.gif)

Reply:
Tested #1574 in IE11, but I can't find any issues. It think the issue might have been solved by refactored code since Jan 18?

![ie11 2019-04-18 13_31_44](https://user-images.githubusercontent.com/1549771/56359367-a711ac80-61e1-11e9-9817-2bbd82f3e3db.gif)